### PR TITLE
Changes for adding instance-resize and puppetizing yum-cron updates

### PR DESCRIPTION
**Pull request by rahulait**
Commit on Mar 08 contains changes for automatic updates.
Commit on Mar 21 contains changes for resize of instances.

### CODE REVIEW

File: moc_openstack/manifests/cronjob.pp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    $rhel7_file_path = "${base_dir}rhel7local-prod.repo"
+    $epel_file_path = "${base_dir}epel7local-prod.repo"
+    $suricata_file_path = "${base_dir}suricata7local-prod.repo"
+    $rhel7_link = "http://${repo_server}/repos/rhel7local-prod.repo"
+    $epel_link = "http://${repo_server}/repos/epel7local-prod.repo"
+    $suricata_link = "http://${repo_server}/repos/suricata7local-prod.repo"
+  } else {
+    $rhel7_file_path = "${base_dir}rhel7local.repo"
+    $epel_file_path = "${base_dir}epel7local.repo"
+    $suricata_file_path = "${base_dir}suricaa7local.repo"
```
Comment (rahulait) [inline moc_openstack/manifests/cronjob.pp:16]: typo: need to fix suricata here

Comment (radonm) [inline moc_openstack/manifests/cronjob.pp:16]: "/bin/sed -i '/enabled/c\enabled = 0 ' /etc/yum.repos.d/epel*.repo" 
would disable the local epel repos as well and this is not what we want. That needs to become 2 lines disabling only the default epel repo files:
"/bin/sed -i '/enabled/c\enabled = 0 ' /etc/yum.repos.d/epel.repo" 
"/bin/sed -i '/enabled/c\enabled = 0 ' /etc/yum.repos.d/epel-testing.repo"

**Comment by radonm** [comment]
some adjustment to yum-cron.conf:
for controller
random_sleep = 3
for compute
random_sleep = 90

**Comment by rahulait** [comment]
Closing this as we will divide the commits in 2 and submit separate pull requests

**Pull request closed without merging**